xocto/localtime.py

```diff
@@ -729,3 +729,20 @@ def period_exceeds_one_year(start_at: datetime_.datetime, end_at: datetime_.date
         one_year_after_start_at = one_year_after_start_at.replace(month=3, day=1)
 
     return tz_unaware_end_at > one_year_after_start_at
+
+
+def parse_date(value: str) -> datetime_.date:
+    """
+    Returns a datetime.date for a given ISO format date string.
+    """
+    return datetime_.date.fromisoformat(value)
+
+
+def parse_dt(value: str) -> datetime_.datetime:
+    """
+    Returns a datetime.datetime for a given ISO format date/time string.
+
+    The date/time must be naive - not include timezone information.
+    """
+    _datetime = datetime_.datetime.fromisoformat(value)
```

Comment:
🐼 Might be worth copying in https://github.com/octoenergy/kraken-core/blob/60a9023bc11b3880e214e0160f3fa433901325a3/src/octoenergy/plugins/territories/aus/meter_readings/gas/supportsite/serializers.py#L152 here, so that this works with all ISO timestamps (the problem is fixed in later versions of python though)

Reply:
Shouldn't be necessary:

> The date/time must be naive - not include timezone information.

So we wouldn't want that to work anyway - at least not at the moment. Maybe this should cope with zoned timestamps, but then really you could/should just use `fromisoformat` directly.